controllers/commentController.js

```diff
@@ -0,0 +1,303 @@
+import db from '../models';
+import utility from '../lib/utility';
+import helper from '../lib/helper';
+
+const {
+  ArticleComment, CommentReply, User, Profiles, Articles
+} = db;
+
+const noArticle = {
+  error: true,
+  message: 'Article does not exist'
+};
+
+const noComment = {
+  error: true,
+  message: 'Comment does not exist'
+};
+
+const noReply = {
+  error: true,
+  message: 'Reply does not exist'
+};
+
```

Comment:
Hi. Could we move these error messages to a helper file and import them instead? Maybe we could call it something like 'errorMessages'. What do you think?

Reply:
That,s fine.